mlflow/server/js/src/model-registry/components/CreateModelForm.tsx

```diff
@@ -7,8 +7,7 @@
 
 import React, { Component } from 'react';
 
-import { Form } from 'antd';
```

Comment:
@hubertzub-db since the PR is pretty small now I'll include this fix as well. Looks like the `CreateModelForm` is using `antd` instead of `@databricks/design-system`, so the theme doesn't get picked up.

| before | after |
| ------ | ----- |
| <img width="593" alt="Screenshot 2023-11-16 at 11 08 49 AM" src="https://github.com/mlflow/mlflow/assets/148037680/79c74975-6a15-4281-9049-fc97e391c9eb"> | <img width="604" alt="Screenshot 2023-11-16 at 11 06 03 AM" src="https://github.com/mlflow/mlflow/assets/148037680/4ad1556a-7ebc-4b30-a40e-d8b881abfbdf"> |

I had to add a `// @ts-expect-error` due to the `ref` prop not existing though, lmk if this is okay, i saw it here as well:
https://github.com/mlflow/mlflow/blob/67df1b5868291b722d7e1ad03a3225d4ac1fa19b/mlflow/server/js/src/experiment-tracking/components/modals/CreateExperimentForm.tsx#L30